src/react-image-lightbox.js

```diff
@@ -1142,6 +1145,14 @@ class ReactImageLightbox extends Component {
 
     inMemoryImage.onerror = errorEvent => {
       this.props.onImageLoadError(imageSrc, srcType, errorEvent);
+
+      // failed to load so set the state loadErrorStatus
+      this.setState(prevState => ({
+        loadErrorStatus: Object.assign({}, prevState.loadErrorStatus, {
```

Comment:
Please use the object rest spread operator here (see other comment for details).